spec/controllers/step_by_step_pages_controller_spec.rb

```diff
@@ -175,6 +198,22 @@ def stub_publishing_api
     stub_any_publishing_api_publish
   end
 
+  def stub_scheduling_permissions
+    stub_user.permissions << "Scheduling"
+  end
+
+  def stub_publishing_api_for_scheduling
+    allow(Services.publishing_api).to receive(:get_content)
+      .and_return(content_item(step_by_step_page))
+
+    stub_default_publishing_api_put_intent
```

Comment:
Stubbing `put_intent` only required for the the scheduling tests, not the unscheduling ones. 
You could follow the pattern of the unscheduling tests and move this line to the `schedule_publishing` method.